tds/src/main/java/thredds/server/ncss/params/NcssParamsBean.java

```diff
@@ -405,6 +407,20 @@ public boolean isValidGridRequest() {
 
   public boolean intersectsTime(FeatureDataset fd, Formatter errs) throws ParseException {
     CalendarDateRange have =  fd.getCalendarDateRange();
+    try {
+      //
+      // if this is an updatable collection, check for a new CalendarDateRange (i.e. new
+      // data available).
+      // The issue is that fd does not get updated with thredds.catalog.InvDatasetFcPoint
+      //
+      // This allows requests to be fulfilled, even if the correct CalendarDateRange is
+      // not available in the catalog. Ideally these should be synced.
+      //
+      UpdateableCollection uc = (UpdateableCollection) fd;
+      have = uc.update();
+    } catch (IOException e) {
```

Comment:
Ok, this should be fixed in the updated PR. Thanks!

On Monday, February 8, 2016, Ryan May <notifications@github.com> wrote:

> In tds/src/main/java/thredds/server/ncss/params/NcssParamsBean.java
> <https://github.com/Unidata/thredds/pull/413#discussion_r52215923>:
>
> > @@ -405,6 +407,20 @@ public boolean isValidGridRequest() {
> >
> >    public boolean intersectsTime(FeatureDataset fd, Formatter errs) throws ParseException {
> >      CalendarDateRange have =  fd.getCalendarDateRange();
> > +    try {
> > +      //
> > +      // if this is an updatable collection, check for a new CalendarDateRange (i.e. new
> > +      // data available).
> > +      // The issue is that fd does not get updated with thredds.catalog.InvDatasetFcPoint
> > +      //
> > +      // This allows requests to be fulfilled, even if the correct CalendarDateRange is
> > +      // not available in the catalog. Ideally these should be synced.
> > +      //
> > +      UpdateableCollection uc = (UpdateableCollection) fd;
> > +      have = uc.update();
> > +    } catch (IOException e) {
>
> Probably need to log an error on the IOException and catch (and swallow)
> any errors from casting to UpdatableCollection.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Unidata/thredds/pull/413/files#r52215923>.
>